src/MSALWrapper/TokenResult.cs

```diff
@@ -87,9 +68,9 @@ public JsonWebToken JWT
         public TimeSpan ValidFor { get; internal set; }
 
         /// <summary>
-        /// Gets the auth type.
+        /// Gets a value indicating whether this token was acquired silently or not.
         /// </summary>
-        public AuthType AuthType { get; internal set; }
+        public bool Silent { get; internal set; }

```

Comment:
I prefer having a bool property as a verb so that the intention is clear. Consider using `IsSilent`.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members